Fixed DRC error SiEPIC1a (id: fares_770)

FaresBadrCA opened the pull request:
Fixed the DRC error about mismatches waveguide widths. 
The DRC error about the adiabatic Y  being smaller than the allowed feature size still exists and I am not sure how to get past it.

lukasc-ubc commented:
Hi,

The errors in your layout are because the grating couplers are not exactly ∆y=127 ∆x=0 apart. In your case the horizontal offset is 600 nm.

<img width="837" alt="image" src="https://github.com/SiEPIC/openEBL-2024-05/assets/15843200/3f85bdbf-7af9-402f-80eb-54b450403cac">

FaresBadrCA commented:
In fixing the DRC error in the last commit, I moved one of the grating couplers horizontally, which introduced a layout error. I moved the output coupler to fix the issue I introduced.
The DRC error called "Si_width" remains. When doing the peer review, I noticed others who use tapers have the same DRC error as well.
